migrate.py

```diff
@@ -129,7 +138,7 @@ def fix_wiki_syntax(self, markup):
         markup = re.sub(r'^ [-\*] ', '* ', markup)
         markup = re.sub(r'\n [-\*] ', '\n* ', markup)
 
-        markup = re.sub(r'\[changeset:"([^"/]+?)(?:/[^"]+)?"]', lambda i: self.convert_revision_id(i.group(1)), markup)
+        markup = re.sub(r'\[changeset:"([^"/]+?)(?:/[^"]+)?"[^\]]*]', lambda i: self.convert_revision_id(i.group(1)), markup)
```

Comment:
handle cases like
[changeset:\"854492e15fe45baccedd1db674f133ff76250dc9/proj\" 854492e/proj]
where the ' 854492e/proj' may be be optionally present.